components/finance/FinanceDateHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@material-ui/core'
+import { createStyles, makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import DateHelper from '../../../lib/dataHelper'
+
+function createData(date: string, value: number) {
+  return { date, value }
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    todayHeader: {
+      borderRadius: 0,
+      color: 'white',
+      margin: 'auto',
+      width: '10.15rem',
+      background: '#255d6d',
+      '&:hover': {
+        background: '#255d6d',
```

Comment:
also this color

Reply:
could have used tailwind if weren't overriding hover behavior above. hence not yet.